File: src/main/java/net/Chidoziealways/everythingjapanese/entity/client/ironbattleaxe/IronBattleAxeRenderState.java
package net.Chidoziealways.everythingjapanese.entity.client.ironbattleaxe;

import net.Chidoziealways.everythingjapanese.entity.custom.IronBattleAxeProjectileEntity;
import net.minecraft.client.renderer.entity.state.EntityRenderState;
import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.api.distmarker.OnlyIn;

@OnlyIn(Dist.CLIENT)
public class IronBattleAxeRenderState extends EntityRenderState {
    public float xRot;
    public float yRot;
    public float shake;
    private IronBattleAxeProjectileEntity entity;
    private float pPartialTick;

    public IronBattleAxeProjectileEntity getEntity() {
        return entity;
    }

    public float getpPartialTick() {
        return pPartialTick;
    }

    public void setEntity(IronBattleAxeProjectileEntity entity) {
        this.entity = entity;
    }
}

File: src/main/kotlin/net/Chidoziealways/everythingjapanese/entity/client/ironbattleaxe/IronBattleAxeRenderState.kt
package net.Chidoziealways.everythingjapanese.entity.client.ironbattleaxe

import net.Chidoziealways.everythingjapanese.entity.custom.IronBattleAxeProjectileEntity
import net.minecraft.client.renderer.entity.state.EntityRenderState
import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.api.distmarker.OnlyIn

@OnlyIn(Dist.CLIENT)
class IronBattleAxeRenderState : EntityRenderState() {
    var xRot: Float = 0f
    var yRot: Float = 0f
    var shake: Float = 0f
    private var entity: IronBattleAxeProjectileEntity? = null
    private val pPartialTick = 0f

    fun getEntity(): IronBattleAxeProjectileEntity? {
        return entity
    }

    fun getpPartialTick(): Float {
        return pPartialTick
    }

    fun setEntity(entity: IronBattleAxeProjectileEntity?) {
        this.entity = entity
    }
}
